pkg/controller/remote/util.go

```diff
@@ -61,17 +60,11 @@ func GetKubeConfigSecret(c client.Client, cluster, namespace string) (*corev1.Se
 	return secret, nil
 }
 
-// DecodeKubeConfigSecret uses the Secret to retrieve and decode the data.
-func DecodeKubeConfigSecret(secret *corev1.Secret) ([]byte, error) {
-	encodedKubeconfig, ok := secret.Data[kubeconfigSecretKey]
+// KubeConfigFromSecret uses the Secret to retrieve the KubeConfig.
+func KubeConfigFromSecret(secret *corev1.Secret) (out []byte, err error) {
```

Comment:
I don't think this needs named return values?

Reply:
Leftover from testing, thanks!